ports/cheriot-rtos/Makefile

```diff
@@ -10,7 +10,7 @@ MICROPY_ROM_TEXT_COMPRESSION ?= 0
 include $(TOP)/py/py.mk
 include $(TOP)/extmod/extmod.mk
 
-CLLVM_DIR ?= ~/cheriot-llvm/
+CLLVM_DIR ?= ~/dev/llvm-project/builds/cheriot-llvm/
```

Comment:
Don't commit changes specific to your build environment

Reply:
done